### ISSUE TITLE
No ChatId object in entities

### ISSUE THREAD

**Issue by dmitvitalii** [kotlin-telegram-bot/kotlin-telegram-bot#176]
**Expected behavior**
Expected ChatId object in com.github.kotlintelegrambot.entities.

**Actual behavior**
There's no such object in a library from jetpack

**To Reproduce**
Add
```
implementation 'io.github.kotlin-telegram-bot.kotlin-telegram-bot:telegram:6.0.2'
```
And use an example from README.md
```
val bot = bot {
        token = API_TOKEN
        dispatch {
            command("start") {
                val result = bot.sendMessage(chatId = ChatId.fromId(message.chat.id), text = "Hi there!")
                result.fold({
                    // do something here with the response
                },{
                    // do something with the error
                })
            }
        }
    }
    bot.startPolling()
```

**Screenshots/Traceback**
<img width="852" alt="Screenshot 2021-02-14 at 21 36 44" src="https://user-images.githubusercontent.com/14320898/107885702-e52aa500-6f0c-11eb-9299-848889e14865.png">
<img width="510" alt="Screenshot 2021-02-14 at 21 36 24" src="https://user-images.githubusercontent.com/14320898/107885704-e78cff00-6f0c-11eb-8826-8245322ce366.png">
<img width="608" alt="Screenshot 2021-02-14 at 21 42 32" src="https://user-images.githubusercontent.com/14320898/107885856-8dd90480-6f0d-11eb-9b4f-047394e12cbb.png">

**Possible solutions**

**Additional context**

**Comment by vjgarciag96** [comment]
Oh true, that's my bad! 

We updated the documentation for the new `ChatId` class but we hadn't published a new version of the library including that class. 

I just released the 6.0.3 version and you should be able to find it there. 

Let me know if you find any problem.

Cheers!

**Comment by dmitvitalii** [comment]
Hi, @vjgarciag96 , found it, thank you!
